packages/vitest/src/node/core.ts

```diff
@@ -609,7 +609,7 @@ export class Vitest {
     await this.report('onPathsCollected', filepaths)
     await this.report('onSpecsCollected', specs.map(
       ([project, file]) =>
-        [{ name: project.getName(), root: project.config.root }, file] as SerializableSpec,
+        [{ name: project.config.name, root: project.config.root }, file] as SerializableSpec,
```

Comment:
This caused a bug duplicating tasks in the UI, it's noticeable in watch mode, but easy to miss if the page is refreshed